github-actions/bazel/setup/configure-bazelrc.ts

```diff
@@ -0,0 +1,26 @@
+import {cpus, totalmem} from 'os';
+import {appendFileSync, readFileSync} from 'fs';
+
+/** Path to the provided bazelrc file. */
+const bazelRcPath = process.env['BAZELRC_PATH'];
+
+if (bazelRcPath === undefined) {
+  throw Error('BAZELRC_PATH environment variable was not provided');
+}
+
+/** Total CPU cores available on the system. */
+const totalCpus = cpus().length;
```

Comment:
Are we sure that this is more correct than the CPUs/RAM calculation from Bazel? My understanding is that these methods were returning wrong information because of the docker VM.